mmdet/ops/conv_ws.py

```diff
@@ -20,7 +19,6 @@ def conv_ws_2d(input,
     return F.conv2d(input, weight, bias, stride, padding, dilation, groups)
 
 
-@CONV_LAYERS.register_module(name='ConvWS')
```

Comment:
It seems this module has been registered elsewhere, and the last build failed. I am not clear why earlier builds succeeded.

Reply:
This is due to the version of mmcv. v0.6.2 works for the current version and v1.0 will work after #3232 is merged.

Reply:
OK. I have merged latest commits into this branch.